UPGRADE-5.0.md

```diff
@@ -26,6 +26,10 @@ The underlying library used for sending emails (Swift Mailer) was discontinued a
       * `.env.$APP_ENV`       committed environment-specific defaults
       * `.env.$APP_ENV.local` uncommitted environment-specific overrides
     * The system run similar index_dev.php if you use `APP_ENV=dev` and `APP_DEBUG=1` in your .env.local file.
+    * Remove Custom shortener API URL from configuration (link_shortener_url variable) and remove `Mautic\CoreBundle\Helper\UrlHelper::buildShortUrl()` method. 
+      * Add the new shortening service in Mautic 5. Reconfiguration is needed after migrating to Mautic 5.
+        * A user reads the documentation to understand how to set it up in the configuration.
```

Comment:
Well, I missed that it wasn't added to the "User facing changes" as I asked a couple of times. Can you please move this to line 7?

Reply:
Ah, I didn't notice that section in markdown file. I've made changes

Reply:
I don't see the changes applied. Have you pushed them

Reply:
Ah, pushed now